docs/src/tutorials/conic/optimal_power_flow.jl

```diff
@@ -0,0 +1,304 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Optimal power flow
+
+# *This tutorial was originally contributed by James Foster (@jd-foster).*
+
+# This tutorial formulates and solves an optimal power flow problem,
+# a much-studied nonlinear problem from the field of electrical engineering.
+# Our particular focus is on obtaining a good estimate of the objective value
+# at the global optimum through the use of semidefinite programming techniques.
+
+# The tutorial highlights JuMP's ability to formulate problems involving
+# complex-valued decision variables and complex matrix cones such as the 
+# [`HermitianPSDCone`](@ref) object.
+# See the [Complex number support](@ref) section of the manual for more details.
+
+# For another example of modeling with complex decision variables
+# see the [Quantum state discrimination](@ref) tutorial.
+
+# ## Required packages
+
+# This tutorial requires the following packages:
+
+using JuMP
+import Ipopt
+import SCS
+import LinearAlgebra
+import SparseArrays
+import DataFrames
+import Test
+
+# ## Formulation
+
+# Optimal power flow problems for electrical transmission typically pose the
+# following question: what is the most cost-effective operation of electricity
+# generators while meeting constraints on the safe limits of network components?
+
+# We will use the 9 _bus_ network test case `case9mod` to explore this problem
```

Comment:
In there, it seems "case9mod" is some standard name but it's only where from. Is it coming from some benchmarking suite ?